.jscsrc

```diff
@@ -1,5 +1,6 @@
 {
 	"preset": "jquery",
 	"maximumLineLength": null,
-	"requireCamelCaseOrUpperCaseIdentifiers": null
+	"requireCamelCaseOrUpperCaseIdentifiers": null,
+	"requireCapitalizedComments": false
```

Comment:
> I dont know why they are commented out. I would e.g. add a text message above the commented code so it will work with the code-style or maybe even completly drop those.

I'll go with the first option.



> Did you test if those test pass in case they are not commented?

Not yet, I'm willing to.